reference/generators/premake.rst

```diff
@@ -56,6 +56,9 @@ For each requirement *conanbuildinfo.premake.lua* file declares the following va
 +---------------------------+------------------------------------------------------+
 | conan_rootpath_XXX        | Abs path to root package folder                      |
 +---------------------------+------------------------------------------------------+
+| conan_frameworks_XXX      | Declared cpp_info.frameworks                         |
```

Comment:
Please add a note somewhere that this doesn't work for packaged frameworks (only system), as discussed in the PR